database/index.js

```diff
@@ -1,18 +1,8 @@
 const mysql = require('mysql');
-const config = require('./config.js');
+const config = require('./config_example.js');
```

Comment:
if this is how you do it, your config is concealed and you can leave it as `config.js`. prev comment was like either or situation. no biggie tho.